src/OpenGL/Extensions/Silk.NET.OpenGL.Extensions.ImGui/ImGuiController.cs

```diff
@@ -16,23 +16,24 @@ public class ImGuiController : IDisposable
         private GL _gl;
         private IView _view;
         private IInputContext _input;
-        private bool _frameBegun;
-
-        // OpenGL objects
-        private uint _vertexArray;
-        private uint _vertexBuffer;
-        private uint _vertexBufferSize;
-        private uint _indexBuffer;
-        private uint _indexBufferSize;
         private Version _glVersion;
-
-        private Texture _fontTexture;
-        private Shader _shader;
-        
         private int _windowWidth;
         private int _windowHeight;
+        private bool _frameBegun;
+        private readonly List<char> _pressedChars = new List<char>();
+        private IKeyboard _keyboard;
 
-        private System.Numerics.Vector2 _scaleFactor = System.Numerics.Vector2.One;
+        private static uint _shaderHandle;
+        private static uint _vertHandle;
+        private static uint _fragHandle;
+        private static int _attribLocationTex;
+        private static int _attribLocationProjMtx;
+        private static int _attribLocationVtxPos;
+        private static int _attribLocationVtxUV;
+        private static int _attribLocationVtxColor;
+        private static uint _vboHandle;
+        private static uint _elementsHandle;
+        private static uint _fontTexture;
```

Comment:
What happens if we have two ImGui controllers in one application? (i.e. two windows)

Remove `static`...